pyct/tests/test_report.py

```diff
@@ -0,0 +1,50 @@
+from unittest.mock import patch
+from pyct.report import report
+
+class TestModule:
+    __version__ = "1.9.3"
+    __file__ = "/Users/kbowen/opt/anaconda3/envs/pyct/lib/python3.7/site-packages/param/__init__.py"
+
+@patch("builtins.print")
+@patch("importlib.import_module")
+def test_report_gives_package_version(mock_import_module, mock_print):
+    module = TestModule()
+    mock_import_module.return_value = module
+    
+    report("param")
+    
+    mock_print.assert_called_with('param=1.9.3                    # /Users/kbowen/opt/anaconda3/envs/pyct/lib/python3.7/site-packages/param')
```

Comment:
Presumably should replace `/Users/kbowen` with something like `/mock` or `/dummy`?

Reply:
I updated these changes